src/main/java/Main.java

```diff
@@ -1,10 +1,18 @@
+import Calculator.Calculator;
+import Calculator.ScientificCalculator;
+
 public class Main {
     public static void main (String[] args) {
         double a = 3;
         double b = 2;
-        System.out.println("Addition: " + a + " + " + b + " = " + Calculator.addition(a, b));
-        System.out.println("Subtraction: " + a + " - " + b + " = " + Calculator.subtraction(a, b));
-        System.out.println("Multiplication: " + a + " * " + b + " = " + Calculator.multiplication(a, b));
-        System.out.println("Division: " + a + " / " + b + " = " + Calculator.division(a, b));
+
+        Calculator calc = new Calculator();
+        System.out.println("Addition: " + a + " + " + b + " = " + calc.addition(a, b));
```

Comment:
please try String.format instead of + concatenation

Reply:
Done